app/src/release/java/org/fossasia/susi/ai/rest/ClientBuilder.java

```diff
@@ -2,6 +2,7 @@
 
 import org.fossasia.susi.ai.helper.PrefManager;
 import org.fossasia.susi.ai.rest.interceptors.TokenInterceptor;
+import org.fossasia.susi.ai.rest.services.SusiService;
```

Comment:
yes it is required. We are using it [here](https://github.com/fossasia/susi_android/pull/646/files#diff-065ddb10ddba248457416ce1087da38bR20). This was the reason why Circle CI build was failing.